src/components/DeleteMediaModal/DeleteMediaModal.tsx

```diff
@@ -0,0 +1,161 @@
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Text,
+  Tr,
+  VStack,
+} from "@chakra-ui/react"
+import {
+  Button,
+  Checkbox,
+  ModalCloseButton,
+} from "@opengovsg/design-system-react"
+import { useState } from "react"
+import { BiImage } from "react-icons/bi"
+
+import { MediaLabels, SelectedMediaDto } from "types/media"
+
+interface DeleteMediaModalProps {
+  selectedMedia: SelectedMediaDto[]
+  mediaLabels: MediaLabels
+  isWriteDisabled: boolean | undefined
+  isOpen: boolean
+  isLoading: boolean
+  onClose: () => void
+  onProceed: (selectedMedia: SelectedMediaDto[]) => void
+}
+
+export const DeleteMediaModal = ({
+  selectedMedia,
+  mediaLabels,
+  isWriteDisabled,
+  isOpen,
+  isLoading,
+  onClose,
+  onProceed,
+}: DeleteMediaModalProps): JSX.Element => {
+  const { singularMediaLabel, pluralMediaLabel } = mediaLabels
+  const [isDeleteChecked, setIsDeleteChecked] = useState(false)
+
+  return (
+    <Modal isOpen={isOpen} onClose={onClose}>
+      <ModalOverlay />
+      <ModalContent>
+        <ModalCloseButton />
+
+        {selectedMedia.length === 1 && (
+          <>
+            <ModalHeader>
+              <Text as="h4" textStyle="h4">
+                Delete {selectedMedia[0].filePath.split("/").pop()}?
+              </Text>
+            </ModalHeader>
+
+            <ModalBody mt="1rem">
+              Are you sure you want to delete this {singularMediaLabel}? If you
+              used this {singularMediaLabel} on any page, site visitors may see
+              a broken {singularMediaLabel}. This cannot be undone.
+            </ModalBody>
+          </>
+        )}
+
+        {selectedMedia.length > 1 && (
+          <>
+            <ModalHeader>
+              <Text as="h4" textStyle="h4">
+                Delete {selectedMedia.length} {pluralMediaLabel}?
+              </Text>
+            </ModalHeader>
+
+            <ModalBody mt="1rem">
+              <VStack spacing="1.5rem">
+                <Text as="p" textStyle="body-1">
+                  Are you sure you want to delete {selectedMedia.length}{" "}
+                  {pluralMediaLabel}?
+                  <br />
+                  <br />
+                  If you used these {pluralMediaLabel} on any page, site
+                  visitors may see a broken {singularMediaLabel}. This cannot be
+                  undone.
+                </Text>
+
+                <Box w="100%">
+                  <Text textStyle="subhead-1">Selected images</Text>
+                  <Text textStyle="caption-2" mt="0.25rem">
+                    {selectedMedia.length} {pluralMediaLabel} will be deleted
+                  </Text>
+                  <TableContainer
+                    mt="0.75rem"
+                    overflowY="scroll"
+                    maxH="14.5rem"
+                  >
+                    <Table variant="simple">
+                      <Tbody borderTop="1px" borderColor="gray.100">
```

Comment:
lets use rem?

Reply:
Wait this needs to be exactly 1 px to match how the other intermediate dividers are defined within the Tbody.